app/controllers/api/v1/agencies_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# API for agencies.  Note that this class is 'read only' for client apps.
+#   Data required for Agencies will be added to the db via rake tasks or other external means.
+class Api::V1::AgenciesController < Api::V1::ApplicationController
```

Comment:
I think for now we should remove the controller actions because we'll be using rake tasks.  If we need routes later on, I'd rather add them in then.  I'd love to see a little documentation added to this effect; could we start a `docs/????.md` file describing caveats and architectural decisions?  I'm not 100% sure what to call it 🤔

Reply:
I'll create one:  technical-decisions.md  (we can always rename it later).

I'd suggest we keep it simple for now and then later you can decide if it needs to be more structured / formalized, etc.
(There are, in fact, some defined structures to us for these. But that's waaay overkill. :-)  )
My experience has been that knowing where to do the search(es) to find an answer where you wonder "why did we do it like this?!?"  is 80% of what you want. (or even 90%).  YMMV